ui/raidboss/data/07-dt/raid/r8s.txt

```diff
@@ -1,9 +1,322 @@
+672.6 "Wind Surge" duration 11.7 #Ability { id: "A891", source: "Howling Blade" }
+676.1 "Twofold Tempest" Ability { id: "A474", source: "Howling Blade" }
+679.8 "Wind Surge" duration 11.7 #Ability { id: "A891", source: "Howling Blade" }
+682.7 "Twofold Tempest" Ability { id: "A474", source: "Howling Blade" }
+686.4 "Wind Surge" duration 11.7 #Ability { id: "A891", source: "Howling Blade" }
+688.6 "Twofold Tempest" Ability { id: "A474", source: "Howling Blade" }
+692.3 "Wind Surge" duration 11.7 #Ability { id: "A891", source: "Howling Blade" }
+699.4 "Bare Fangs" Ability { id: "A475", source: "Howling Blade" }
+
+# Champion's Circuit
+711.0 "Champion's Circuit" Ability { id: ["A477", "A478"], source: "Howling Blade" }
+711.0 "Gleaming Barrage" #Ability { id: "A476", source: "Gleaming Fang" }
+711.7 "--sync--" Ability { id: "A47D", source: "Howling Blade" }
+715.4 "Gleaming Barrage" #Ability { id: "A476", source: "Gleaming Fang" }
+716.1 "Champion's Circuit" Ability { id: "A482", source: "Howling Blade" }
+719.8 "Gleaming Barrage" #Ability { id: "A476", source: "Gleaming Fang" }
+720.4 "Champion's Circuit" Ability { id: "A482", source: "Howling Blade" }
+724.1 "Gleaming Barrage" #Ability { id: "A476", source: "Gleaming Fang" }
+724.7 "Champion's Circuit" Ability { id: "A482", source: "Howling Blade" }
+728.5 "Gleaming Barrage" #Ability { id: "A476", source: "Gleaming Fang" }
+728.9 "Champion's Circuit" Ability { id: "A482", source: "Howling Blade" }
+738.7 "Quake III" Ability { id: "A45A", source: "Howling Blade" }
+752.7 "Ultraviolent Ray 3" Ability { id: "A45C", source: "Howling Blade" }
+752.7 "Gleaming Beam" Ability { id: "A45E", source: "Gleaming Fang" }
+763.7 "Twinbite" Ability { id: "A4CD", source: "Howling Blade" }
+778.9 "Rise of the Hunter's Blade" Ability { id: "A82C", source: "Howling Blade" }
+
+# Lone Wolf
+784.1 "Lone Wolf's Lament" Ability { id: "A483", source: "Howling Blade" }
+794.2 "Prowling Gale" Ability { id: "A485", source: "Howling Blade" }
+802.8 "Prowling Gale (Towers)" Ability { id: "A487", source: "Howling Blade" }
+812.4 "Hero's Blow" Ability { id: ["A460", "A462"], source: "Howling Blade" }
+824.6 "Ultraviolent Ray 4" Ability { id: "A45C", source: "Howling Blade" }
+824.6 "Gleaming Beam" Ability { id: "A45E", source: "Gleaming Fang" }
+839.4 "Howling Eight" Ability { id: "AA02", source: "Howling Blade" }
+840.5 "Howling Eight 1" Ability { id: "AA03", source: "Howling Blade" }
+841.5 "Howling Eight 2" Ability { id: "AA04", source: "Howling Blade" }
+842.4 "Howling Eight 3" Ability { id: "AA05", source: "Howling Blade" }
+843.3 "Howling Eight 4" Ability { id: "AA06", source: "Howling Blade" }
+844.0 "Howling Eight 5" Ability { id: "AA07", source: "Howling Blade" }
+844.4 "Howling Eight 6" Ability { id: "AA08", source: "Howling Blade" }
+844.8 "Howling Eight 7" Ability { id: "AA09", source: "Howling Blade" }
+846.6 "Howling Eight 8" Ability { id: "AA0A", source: "Howling Blade" }
```

Comment:
Remove/comment out the sync on the 8 hits to prevent potential timeline desyncs, here and below.

Reply:
Added.